### PR TITLE
Use tabs by default

### LINKED ISSUE #1: Use tabs by default
Inspired by:

Discussion: https://twitter.com/Rich_Harris/status/1541761871585464323?t=x5U__3u7sO8lzoopxMCBaQ&s=19

### LINKED ISSUE #3: Only include overrides
Should we only include the specific overrides that we want to implement? That would make the config easier to understand from the perspective of what's different/important to LPL

### PR DESCRIPTION
**Pull request by chawes13**
Resolves #1 and #3

### CODE REVIEW

File: index.json [hunk 1]
```diff
@@ -1,17 +1,6 @@
...
   "singleQuote": true,
-  "jsxSingleQuote": false,
-  "trailingComma": "es5",
-  "bracketSpacing": true,
-  "jsxBracketSameLine": false,
-  "arrowParens": "always",
-  "requirePragma": false,
-  "insertPragma": false,
-  "proseWrap": "preserve",
-  "htmlWhitespaceSensitivity": "css",
```
Comment (chawes13) [inline index.json:15]: Removing the options that match the defaults makes it easier for developers to understand the overrides that are specific to LPL

File: index.json [hunk 2]
```diff
@@ -1,17 +1,6 @@
 {
-  "printWidth": 80,
-  "tabWidth": 2,
-  "useTabs": false,
+  "useTabs": true,
```
Comment (chawes13) [inline index.json:2]: Tabs take up less space (in terms of bytes) and most editors allow for configuration of custom width per tab, making them more accessible.

Devs can still write code with spaces and it'll get transformed by the precommit hooks.

File: package.json [hunk 3]
```diff
@@ -1,8 +1,11 @@
...
   "name": "@launchpadlab/prettier-config",
-  "version": "1.0.0",
+  "version": "2.0.0",
   "main": "index.json",
   "repository": "https://github.com/launchpadlab/prettier-config",
   "author": "dpikt",
-  "license": "MIT"
+  "license": "MIT",
+  "peerDependencies": {
+    "prettier": "^1.0 || ^2.0 || ^3.0"
```
Comment (chawes13) [inline package.json:9]: This matches our setup in `esling-config`. It also is a slightly safer configuration, now that we're only including LPL-specific overrides. If/when the next major change is released, we can review the options and ensure that no changes need to be made to accommodate v4.

File: index.json (hunk 2 above)
Comment (bhennes2) [reply index.json:2]: Is the default `tabWidth` now `2`?

**Review by bhennes2 — APPROVED** [review]
LGTM! Had a clarifying question on default tab width (I prefer `2` FWIW)

File: index.json (hunk 2 above)
Comment (chawes13) [reply index.json:2]: Yep! https://prettier.io/docs/en/options.html#tab-width

**Pull request merged**